README.md

```diff
@@ -21,9 +21,15 @@ Once installed the Prometheus Operator provides the following features:
 For an introduction to the Prometheus Operator, see the initial [blog
 post](https://coreos.com/blog/the-prometheus-operator.html).
 
-See [kube-prometheus](https://github.com/coreos/kube-prometheus) for a
-collection of resources that can be used to start monitoring Kubernetes
-and applications running on top of it within minutes.
+## [kube-prometheus](https://github.com/coreos/kube-prometheus)
```

Comment:
"## prometheus-operator vs. kube-prometheus" would greatly help clarify the purpose of this section